shakenfist/node.py

```diff
@@ -0,0 +1,112 @@
+from functools import partial
+import time
+
+from shakenfist import baseobject
+from shakenfist import etcd
+from shakenfist import logutil
+from shakenfist import util
+
+
+LOG, _ = logutil.setup(__name__)
+
+
+class Node(baseobject.DatabaseBackedObject):
+    object_type = 'node'
+    current_version = 2
+    state_targets = {
+        None: ('created', 'error', 'missing'),
```

Comment:
state_targets should use the defined constants

Reply:
Good catch, done.